src/administration/components/AdminPanel.jsx

```diff
@@ -179,6 +191,7 @@ class Admin extends Component {
                         </tbody>
                     </table>
                 </div>
+                {this.state.dataPerPage > this.state.volunteers.length ? <p></p> : pagination}
```

Comment:
Ooooh, hacky hacks! The idiomatic React way would look something like:
```jsx
<Paginator data={paginatedData} itemsPerPage={10} />
```
1. No need to extract it into a local variable (like with the `const` declaration above).
1. If it doesn't need to be displayed, you can use [more elegenat conditional rendering expressions](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator), or you can ever return `null` from the paginator's `render()` method if there is no data.